CLI.md

```diff
@@ -719,6 +722,77 @@ Options:
 | --keystoredir \<path\> | specify the wallet directory \<"global"\|"local"\|custom\> |
 | --chain \<name\> | chain name from "chain.json" |
 
+#### iexec account approve
+
+approve spender to spend up to amount of RLC from your iExec account (default unit nRLC)
+
+Usage:
+
+```sh
+iexec account approve <amount> <spender> [unit] [options]
+```
+
+Options:
+
+| option | description |
+| --- | --- |
+| --raw | use raw output |
+| --quiet | stop prompting updates |
+| --password \<password\> | password used to encrypt the wallet (unsafe) |
+| --wallet-file \<walletFileName\> | specify the name of the wallet file to use |
+| --wallet-address \<walletAddress\> | specify the address of the wallet to use |
+| --keystoredir \<path\> | specify the wallet directory \<"global"\|"local"\|custom\> |
+| --chain \<name\> | chain name from "chain.json" |
+| --gas-price \<amount unit...\> | set custom gas price for transactions (default unit wei) |
+| --confirms \<blockCount\> | set custom block count to wait for transactions confirmation (default 1 block) |
+
+#### iexec account allowance
+
+checks the amount of allowance approved for the specified spender to use your iExec account (specify --user to see another user's allowance)
```

Comment:
Minor: To match other descriptions

```suggestion
check the amount of allowance approved for the specified spender to use your iExec account (specify --user to see another user's allowance)
```

Reply:
Just noticing it is different from the message initially written on the Jira ticket description. Are we fine with that?

Reply:
yes, the message was adjusted to clarify the use of the --user option, making it more explicit compared to the original in Jira